### PR TITLE
Add unreleased docker release notes

### PR DESCRIPTION
**Pull request by michaellzc**
## Test plan

docs

### CODE REVIEW

File: doc/admin/updates/docker_compose.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ Each section comprehensively describes the steps needed to upgrade, and any manu
 
 <!-- GENERATE UPGRADE GUIDE ON RELEASE (release tooling uses this to add entries) -->
 
+## Unreleased
+
+We made a number of changes to our build-in postgres databases (the `pgsql`, `codeintel-db`, and `codeinsights-db` container)
```
Comment (michaellzc) [inline doc/admin/updates/docker_compose.md:15]:
```suggestion
We made a number of changes to our built-in postgres databases (the `pgsql`, `codeintel-db`, and `codeinsights-db` container)
```

File: doc/admin/updates/docker_compose.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ Each section comprehensively describes the steps needed to upgrade, and any manu
 
 <!-- GENERATE UPGRADE GUIDE ON RELEASE (release tooling uses this to add entries) -->
 
+## Unreleased
+
+We made a number of changes to our built-in postgres databases (the `pgsql`, `codeintel-db`, and `codeinsights-db` container)
+
+- **CAUTION** Add the ability to customize postgres server configuration. If you have customized the config in any way, you should copy your changes to the added `postgresql.conf` file [sourcegraph/deploy-sourcegraph-docker#792](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/792)
```
Comment (michaellzc) [inline doc/admin/updates/docker_compose.md:17]:
```suggestion
- **CAUTION** Add the ability to customize postgres server configuration. If you have customized the config in any way, you should copy your changes to the added `postgresql.conf` files [sourcegraph/deploy-sourcegraph-docker#792](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/792)
```

File: doc/admin/updates/docker_compose.md [hunk 3]
```diff
@@ -10,6 +10,14 @@ Each section comprehensively describes the steps needed to upgrade, and any manu
 
 <!-- GENERATE UPGRADE GUIDE ON RELEASE (release tooling uses this to add entries) -->
 
+## Unreleased
+
+We made a number of changes to our built-in postgres databases (the `pgsql`, `codeintel-db`, and `codeinsights-db` container)
+
+- **CAUTION** Add the ability to customize postgres server configuration. If you have customized the config in any way, you should copy your changes to the added `postgresql.conf` files [sourcegraph/deploy-sourcegraph-docker#792](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/792)
+- Increase the minimal memory requirement of `pgsql` and `codeintel-db` from `2GB` to `4GB`
+- `codeinsights-db` now uses vanilla postgres image [sourcegraph/deploy-sourcegraph-docker#780](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/780)
```
Comment (michaellzc) [inline doc/admin/updates/docker_compose.md:19]:
```suggestion
- `codeinsights-db` container now uses vanilla postgres image [sourcegraph/deploy-sourcegraph-docker#780](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/780)
```

File: doc/admin/updates/docker_compose.md [hunk 4]
```diff
@@ -10,6 +10,14 @@ Each section comprehensively describes the steps needed to upgrade, and any manu
 
 <!-- GENERATE UPGRADE GUIDE ON RELEASE (release tooling uses this to add entries) -->
 
+## Unreleased
+
+We made a number of changes to our built-in postgres databases (the `pgsql`, `codeintel-db`, and `codeinsights-db` container)
+
+- **CAUTION** Add the ability to customize postgres server configuration. If you have customized the config in any way, you should copy your changes to the added `postgresql.conf` files [sourcegraph/deploy-sourcegraph-docker#792](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/792)
```
Comment (caugustus-sourcegraph) [inline doc/admin/updates/docker_compose.md:17]:
```suggestion
- **CAUTION** Added the ability to customize postgres server configuration by mounting external configuration files. If you have customized the config in any way, you should copy your changes to the added `postgresql.conf` files [sourcegraph/deploy-sourcegraph-docker#792](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/792)
```

File: doc/admin/updates/docker_compose.md [hunk 5]
```diff
@@ -10,6 +10,14 @@ Each section comprehensively describes the steps needed to upgrade, and any manu
...
+- Increase the minimal memory requirement of `pgsql` and `codeintel-db` from `2GB` to `4GB`
```
Comment (caugustus-sourcegraph) [inline doc/admin/updates/docker_compose.md:18]:
```suggestion
- Increased the minimal memory requirement of `pgsql` and `codeintel-db` from `2GB` to `4GB`
```

File: doc/admin/updates/docker_compose.md [hunk 6]
```diff
@@ -10,6 +10,14 @@ Each section comprehensively describes the steps needed to upgrade, and any manu
...
+- `codeinsights-db` container now uses vanilla postgres image [sourcegraph/deploy-sourcegraph-docker#780](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/780)
```
Comment (caugustus-sourcegraph) [inline doc/admin/updates/docker_compose.md:19]:
```suggestion
-`codeinsights-db` container no longer uses TimescaleDB and is now based on the standard Postgres image [sourcegraph/deploy-sourcegraph-docker#780](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/780). Metrics scraping is also enabled.
```

File: doc/admin/updates/docker_compose.md [hunk 7]
```diff
@@ -10,6 +10,14 @@ Each section comprehensively describes the steps needed to upgrade, and any manu
 
 <!-- GENERATE UPGRADE GUIDE ON RELEASE (release tooling uses this to add entries) -->
 
+## Unreleased
+
+We made a number of changes to our built-in postgres databases (the `pgsql`, `codeintel-db`, and `codeinsights-db` container)
+
+- **CAUTION** Added the ability to customize postgres server configuration by mounting external configuration files. If you have customized the config in any way, you should copy your changes to the added `postgresql.conf` files [sourcegraph/deploy-sourcegraph-docker#792](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/792)
+- Increased the minimal memory requirement of `pgsql` and `codeintel-db` from `2GB` to `4GB`
+-`codeinsights-db` container no longer uses TimescaleDB and is now based on the standard Postgres image [sourcegraph/deploy-sourcegraph-docker#780](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/780). Metrics scraping is also enabled.
```
Comment (michaellzc) [inline doc/admin/updates/docker_compose.md:19]:
```suggestion
-`codeinsights-db` container no longer uses TimescaleDB and is now based on the standard Postgres image [sourcegraph/deploy-sourcegraph-docker#780](https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/780). Metrics scraping is also enabled. You may now use external Postgres deployment `codeinsights-db`
```

@caugustus-sourcegraph should we mention user can host codeinsights-db elsewhere?

Comment (caugustus-sourcegraph) [reply doc/admin/updates/docker_compose.md:19]: That should be covered by the general release notes (I know Code Insights was working on a release advisory). These update docs are specifically for "things that might break your docker-compose setup".

**Review by caugustus-sourcegraph — APPROVED** [review]

**Pull request merged**